### DISCUSSION TITLE
How to access the event that triggered a workflow

### DISCUSSION THREAD

**Discussion by paramt** [community/community#25519] [Actions]
<p>I have a workflow that is triggered each time an issue is opened. How can I access the issue number of the issue that triggered the workflow? I saw someone using </p>
<pre><code>${{ github.event.pull_request.number }}
</code></pre>
<p>on a forum, but I couldn’t find an equivalent syntax for issues on the docs. <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions#github-context" rel="nofollow noopener">Here</a> it lists the github.event context but I can’t find any specific info on issues.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>You can check the context content, follow the structure to get the issue number, code as below:</p>
<pre><code>echo ${{ github.event.issue.number }}
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/68d241c9484617e65920013cbaf4e7818163007a.png" data-download-href="/uploads/short-url/eXi85lAPT0gdSGqXqiWGBggWob0.png?dl=1" title="issuenumber.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181091696-3f322901-bce1-4685-bef2-954b4c8e2136.png" alt="issuenumber.png" data-base62-sha1="eXi85lAPT0gdSGqXqiWGBggWob0" width="593" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/68d241c9484617e65920013cbaf4e7818163007a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">issuenumber.png</span><span class="informations">649×547 26.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
